Update datadog docs (#2478)
TykTechnologies/tyk-docs: tyk-docs/content/frequently-asked-questions/datadog-logs-showup-as-errors.md

---
date: 2017-03-27T16:30:52+01:00
title: Datadog logs show up under the error status
menu:
  main:

>     parent: "Frequently Asked Questions"

Author: This is not part of the pump configuration. This has to do with datadog agent scraping logs and those logs not being parsed properly by dd

>     parent: "Frequently Asked Questions"

Reviewer: ok, so please put it under troubleshooting/tyk-pump/

>     parent: "Frequently Asked Questions"

Author: I think you miss understand this is not relevant to pump. It scraps all container logs.

>     parent: "Frequently Asked Questions"

Reviewer: ok gotcha.I think we need to make this clearer. 
his FAQ's title is about getting errors `Tyk logs scraped with Datadog agent show up as errors under Datadog` - maybe it should be `Send Tyk container logs to Datadogs `

>     parent: "Frequently Asked Questions"

Author: The question that the user would be asking is `why are tyk logs showing up as errors in my datadog logs` the datadog agent integration is going to pick up the tyk container logs by default and those by default will show up as errors. So maybe we can do `tyk container logs show up under the error status in Datadog logs`

>     parent: "Frequently Asked Questions"

Reviewer: ok, now i remember, I thought I disabled the collection of the logs in my configuration doc, but maybe it was just something i did in my test env

weight: 0 
---

> To allow Datadog to read the Tyk logs correctly. We need to create a pipeline to process the data and allow Datadog to understand that data.

Reviewer: Please add something to clarify that you need a running tyk deployment that already sends data to datadog (with Datadog Agent)

To do that, we need to access the `/logs/pipelines` path on your datadog web application. This will take us to the pipeline configuration page. In here, we will create a new pipeline. For the filter section, use `Service:tyk-*` this will capture logs for any of the Tyk related services.

{{< img src="/img/faq/datadog-logs-showup-as-errors/create-pipeline.png" alt="Create Datadog Logs Pipeline to process Tyk services' logs" >}}

> Next we will need to add processor to the pipeline. Select the Grok Parser processor type, give it a name and click on the `Parse My Logs` button and `Create` .

Reviewer: It is not clear that you need to expand the pipeline that you just created and there you can click to create the parser 

<img width="1315" alt="image" src="https://user-images.githubusercontent.com/3155222/232598977-31126fde-cbbb-463d-9b31-27d4a15d5571.png">

---
date: 2017-03-27T16:30:52+01:00

> title: Datadog logs show up under the error status

Author: ```suggestion
title: Tyk logs scraped with Datadog agent show up as errors under Datadog logs 
```

> title: Tyk logs scraped with Datadog agent show up as errors under Datadog logs 

Reviewer: ```suggestion
title: All tyk container logs show up under the error status in Datadog logs
```

menu:
  main:
    parent: "Frequently Asked Questions"
weight: 0 
---

> To allow Datadog Agent to scrape the logs of your Tyk deployment correctly.

Reviewer: ```suggestion
With Datadog you can view logs of all your Tyk components.
To allow Datadog Agent to scrape the logs of your Tyk deployment correctly you need to create a pipeline in Datadog, to process and underst and this data.
```

> We need to create a pipeline to process the data and allow Datadog to understand that data.

Reviewer: ```suggestion
```
